scripts/build-controller-image.sh

```diff
@@ -11,6 +11,11 @@ DOCKERFILE=${DOCKERFILE:-"$DOCKERFILE_PATH"}
 LOCAL_MODULES=${LOCAL_MODULES:-"false"}
 BUILD_DATE=$(date +%Y-%m-%dT%H:%M)
 QUIET=${QUIET:-"false"}
+CONTAINER_OS=${CONTAINER_OS:-"$(go env GOOS)"}
+
+if [ "$CONTAINER_OS" == "darwin" ]; then
+  CONTAINER_OS="linux"
+fi
```

Comment:
Most of this code is run by `test-infra` repository pipelines. Are you using darwin instances to build images?

Reply:
I sometimes use Apple's M1 Mac instance to build and test images.
I think there are quite a lot of Apple M1 Mac users out there, so this should be useful.